From: Modestas Vainius <modax@debian.org>
Date: Wed, 13 Apr 2016 16:40:33 +0200
Subject: don't set the install RPATH to LIB_INSTALL_DIR don't set the install
 RPATH to LIB_INSTALL_DIR if LIB_INSTALL_DIR is one of the standard system
 link directories, like e.g. /usr/lib (which are listed in
 CMAKE_PLATFORM_IMPLICIT_LINK_DIRECTORIES,
 which is set in Modules/Platform/UnixPaths.cmake). . Also drop custom
 INSTALL_RPATHs in kdewidgets/CMakeLists.txt.

---
 kdewidgets/CMakeLists.txt | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/kdewidgets/CMakeLists.txt b/kdewidgets/CMakeLists.txt
index 5153601..b88b12f 100644
--- a/kdewidgets/CMakeLists.txt
+++ b/kdewidgets/CMakeLists.txt
@@ -51,7 +51,6 @@ if(QT_QTDESIGNER_FOUND)
                                      INSTALL_RPATH_USE_LINK_PATH TRUE
                                      SKIP_BUILD_RPATH TRUE
                                      BUILD_WITH_INSTALL_RPATH TRUE
-                                     INSTALL_RPATH ${LIB_INSTALL_DIR}
                          )
   endif(NOT WIN32)
 
@@ -81,7 +80,6 @@ if(QT_QTDESIGNER_FOUND)
                                           INSTALL_RPATH_USE_LINK_PATH TRUE
                                           SKIP_BUILD_RPATH TRUE
                                           BUILD_WITH_INSTALL_RPATH TRUE
-                                          INSTALL_RPATH ${LIB_INSTALL_DIR}
                            )
     endif(NOT WIN32)
 
@@ -116,7 +114,6 @@ if(QT_QTDESIGNER_FOUND)
                                      INSTALL_RPATH_USE_LINK_PATH TRUE
                                      SKIP_BUILD_RPATH TRUE
                                      BUILD_WITH_INSTALL_RPATH TRUE
-                                     INSTALL_RPATH ${LIB_INSTALL_DIR}
                          )
   endif(NOT WIN32)
 
@@ -152,7 +149,6 @@ if(QT_QTDESIGNER_FOUND)
                                      INSTALL_RPATH_USE_LINK_PATH TRUE
                                      SKIP_BUILD_RPATH TRUE
                                      BUILD_WITH_INSTALL_RPATH TRUE
-                                     INSTALL_RPATH ${LIB_INSTALL_DIR}
                              )
      endif(NOT WIN32)
 
